oss_src/process/process_win.cpp

```diff
@@ -75,6 +110,34 @@ bool process::launch(const std::string &cmd,
 
   logstream(LOG_INFO) << "Launched process with pid: " << m_pid << std::endl;
 
+  // Wait up to 100 milliseconds before querying the process for it's status.
+  DWORD dwMillisec = 100;
+  DWORD dwWaitStatus = WaitForSingleObject(m_proc_handle, dwMillisec );
+
+  if(dwWaitStatus == WAIT_FAILED) {
+    auto err = GetLastError();
+    logstream(LOG_WARNING) << "Error in WaitForSingleObject after CreateProcess: "
+                           << get_last_err_str(err) << std::endl;
+  }
```

Comment:
It seemed 100ms caught the errors I was seeing in python not starting or library import errors, which seemed to be most of the problems.  Balancing this is that we don't want to wait so long that the user notices it.  This seemed like a good compromise.